process/subgroups.md

```diff
@@ -0,0 +1,68 @@
+# Subgroups
+
+Subgroups of the CG are created when one (or both) of the following apply:
+
+- An area of interest that needs a lot of discussion. Creating a subgroup with regular meetings provides more bandwidth for this discussion.
+- An area of interest is only relevant to a small group of participants. In these cases, subgroup meetings give a more focused venue. This way, participants don’t have concerns about wasting the CG’s time with something that the rest of the CG is not engaged in.
+
+To create a subgroup, you should add a poll to a CG meeting agenda with at least 7 days notice. As part of the poll, you should provide information about who will be chairing the subgroup.
+
+## Active Subgroups
```

Comment:
It might be worthwhile to at least mention debugging here, although you are right that calling it an "active" subgroup is a bit of a stretch :) 
I would still like to encourage people to post on the repo if they want to talk about debugging though. Maybe we could list it, but instead of a link to agendas, say something like "no meetings currently scheduled, but discussion or agenda suggestions are welcome"

Reply:
Added